pkg/controller/machinehealthcheck/machinehealthcheck_controller.go

```diff
@@ -38,6 +39,31 @@ const (
 	remediationStrategyReboot     = mhcv1beta1.RemediationStrategyType("reboot")
 	timeoutForMachineToHaveNode   = 10 * time.Minute
 	machineNodeNameIndex          = "machineNodeNameIndex"
+	controllerName                = "machinehealthcheck-controller"
+
+	// Event types
+	// eventRemediationRestricted is emitted in case when machine remediation
+	// is restricted by remediation circuit shorting logic
+	eventRemediationRestricted string = "MHCRemediationRestricted"
```

Comment:
This is a bit of a nit, but I'd prefer to remove the `MHC` prefix on these. I don't think that means anything to users, and the events are already associated with a MachineHealthCheck resource.

I also don't see why we wouldn't export these. They seem like one of the few things in this package that might actually be useful externally.

Reply:
fixed